README.md

```diff
@@ -248,12 +248,12 @@ behavior:
   # Determines the text icon to display next to "liked" Spotify items, such as
   # liked songs and albums, or followed artists. Can be any length string.
   # These icons require a patched nerd font.
```

Comment:
Looks like the previous icons required a patched nerd font, which I must have overlooked